buildspec.yml

```diff
@@ -41,21 +41,27 @@ phases:
       - export IMAGE_TAG="${GITHUB_BRANCH}.${COMMIT_HASH}"
       - echo "GITHUB_BRANCH=${GITHUB_BRANCH} COMMIT_HASH=${COMMIT_HASH} IMAGE_TAG=${IMAGE_TAG}"
       - echo "creating a clean environment"
-      - make clean dev
+      - make setup
+      - echo "linting code"
+      - make lint
```

Comment:
Why not also use the django `./manage.py check` as well in the CI ?

Reply:
just forgot to add it, btw (as you said) it's part of the default django testrunner: https://docs.djangoproject.com/en/3.1/topics/testing/advanced/#using-different-testing-frameworks

Reply:
The Django default test runner doesn't produce junit reports, I see three options:
- [ ] use xmlrunner instead https://github.com/xmlrunner/unittest-xml-reporting#django-support or
- [ ] use pytest https://docs.pytest.org/en/stable/usage.html#creating-junitxml-format-files
- [ ] live without junit reports

I could live without junit reports, the django testrunner gives a nice summary about failing tests in the end. I personally would avoid pytest and try to stick to a django-conform test-runner. So for me option 1 or 3

Reply:
I find the Codebuild Test report (provided by the junit report) not very helpful in case of error, therefore I suggest option 3.